BO4E/ENUM/Handelsunstimmigkeitsgrund.cs

```diff
@@ -0,0 +1,35 @@
+using BO4E.meta;
+
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Begründung der Korrektheit der Rechnung oder des Lieferscheins (COMDIS SG3 AJT 4465)
+    /// </summary>
+    [NonOfficial(NonOfficialCategory.MISSING)]
+    public enum Handelsunstimmigkeitsgrund
+    {
+        /// <summary> ANMDELDUNG_BESTAETIGT</summary>
+        /// <remarks>Z58</remarks>
+        ANMDELDUNG_BESTAETIGT,
```

Comment:
das hatte mich schon bei Annikas REMADV-PR gewundert: In den >2022-10-01 AHBs sind das alles EBDs. Vllt wäre es schlau, die ENUMS direkt schon so auszulegen, dass sie zusätzlich zum aktuell gültigen AHB Value Pool *auch* die möglichen Ausgangswerte der EBDs S_0108, S_0109 und E_0504 abbilden.

Ich verstehe aber, wenn das gerade zu viel Arbeit ist. Vllt ein Todo-Kommentar?